tests/guarded_transmute_pod_vec/mod.rs

```diff
@@ -1,19 +1,20 @@
 #![cfg(feature = "std")]
 
 
-use safe_transmute::{Error, ErrorReason, GuardError, guarded_transmute_pod_vec};
+use safe_transmute::{Error, ErrorReason, GuardError, safe_transmute_vec, SingleManyGuard};
 use self::super::{LeToNative, aligned_vec};
 
+type G = SingleManyGuard;
 
 #[test]
 fn too_short() {
-    assert_eq!(guarded_transmute_pod_vec::<u16>(aligned_vec::<u16>([].as_ref())),
+    assert_eq!(safe_transmute_vec::<u16, G>(aligned_vec::<u16>([].as_ref())),
```

Comment:
Module name doesn't match tested funxion – are we letting this slide 'till the test reorganisation?